docs/source/examples/mg-mode-part-i.ipynb

```diff
@@ -0,0 +1,926 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7fd19aa6f780>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "root_universe.plot(center=(0., 0., 0.), width=(3 * 21.42, 3 * 21.42), pixels=(500, 500),\n",
+    "                   color_by='material')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "OK, it looks pretty good, lets go ahead and write the file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# Create Geometry and set root Universe\n",
+    "geometry = openmc.Geometry()\n",
+    "geometry.root_universe = root_universe\n",
+    "\n",
+    "# Export to \"geometry.xml\"\n",
+    "geometry.export_to_xml()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can now create the tally file information. The tallies will be set up to  give us the pin powers in this notebook. We will do this with a mesh filter, with one mesh cell per pin."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "tallies_file = openmc.Tallies()\n",
+    "\n",
+    "# Instantiate a tally Mesh\n",
+    "mesh = openmc.Mesh()\n",
+    "mesh.type = 'regular'\n",
+    "mesh.dimension = [17 * 2, 17 * 2]\n",
+    "mesh.lower_left = [-32.13, -10.71]\n",
+    "mesh.upper_right = [+10.71, +32.13]\n",
+    "\n",
+    "# Instantiate tally Filter\n",
+    "mesh_filter = openmc.MeshFilter(mesh)\n",
+    "\n",
+    "# Instantiate the Tally\n",
+    "tally = openmc.Tally(name='mesh tally')\n",
+    "tally.filters = [mesh_filter]\n",
+    "tally.scores = ['fission']\n",
+    "\n",
+    "# Add tally to collection\n",
+    "tallies_file.append(tally)\n",
+    "\n",
+    "# Export all tallies to a \"tallies.xml\" file\n",
+    "tallies_file.export_to_xml()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With the geometry and materials finished, we now just need to define simulation parameters for the `settings.xml` file. Note the use of the `energy_mode` attribute of our `settings_file` object. This is used to tell OpenMC that we intend to run in multi-group mode instead of the default continuous-energy mode. If we didn't specify this but our cross sections file was not a continuous-energy data set, then OpenMC would complain."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# OpenMC simulation parameters\n",
+    "batches = 1000\n",
+    "inactive = 100\n",
+    "particles = 5000\n",
```

Comment:
I agree with @smharper that we should use fewer batches/particles at the risk of the results not looking quite as good (that's just MC). In fact, we might want to devise and enforce a rule that no notebook can take more than say, a 90 seconds to run, or something like that (and one or more of the current notebooks may already break this rule). I think this is important for the following reasons:

(1) The longer it takes to run a notebook, the larger the burden on developers who *should* be running the notebooks before submitting PRs to confirm that they aren't affected.
(2) When developers do run the notebooks, they will be inclined to reduce the number of batches/particles anyway, and may accidentally commit these slight adjustments. The parameters should be designed such that there is little incentive for a developer to change them.
(3) IMO we should aim to include the notebooks as part of the test suite someday (even if they aren't run on Travis) to better automate QA for our development workflow.

Just my two cents.